content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods/before_scheduled_date.rst

```diff
@@ -0,0 +1,118 @@
+
+Once selected, a new :guilabel:`Reserve before scheduled date` field appears below. From this field,
+the number of :guilabel:`days before` and :guilabel:`days before when starred` can be changed from
+the default `0`.
+
+Changing the :guilabel:`days before` value changes the maximum number of days before a scheduled
+date that products should be reserved.
+
+Changing the :guilabel:`days before when starred` value changes the maximum number of days before
+a scheduled date that products should be reserved for starred (favorited) transfers.
+
+.. example::
+   In the image below, the :guilabel:`days before` value is set to `2` days before, and the
+   :guilabel:`days before when starred` value is set to `3`.
+
+   This means that products will be reserved two days before the scheduled delivery date for normal
+   orders, and three days before the scheduled delivery date for starred (favorited) transfers.
+
+   .. image:: before_scheduled_date/before-scheduled-date- days-before.png
+      :align: center
+      :alt: Reserve before scheduled date field with set numerical values.
+
+   This is the configuration applied for the following workflow below.
+
+Edit product form
+-----------------
+
+Before beginning the workflow, first add a *customer lead time* to the product that will be sold. To
+do so, navigate to :menuselection:`Products --> Products`, and select the desired product.
+
+Click the :guilabel:`Inventory` tab, and under the :guilabel:`Logistics` section, change the value
+in the :guilabel:`Customer Lead Time` field to `5` days. This sets the scheduled delivery date for
+this specific product to five days after the creation date of the sales order.
+
+.. image:: before_scheduled_date/before-scheduled-date-customer-lead-time.png
+   :align: center
+   :alt: Product form with customer lead time set in Inventory tab.
+
+Workflow
+========
+
+To see the *Before scheduled date* reservation method in action, create a new :abbr:`SO (sales
+order)` by navigating to :menuselection:`Sales app --> New`.
+
+Add a customer in the :guilabel:`Customer` field, then, in the :guilabel:`Order Lines` tab, click
+:guilabel:`Add a product`, and select a product to add to the quotation from the drop-down menu.
+Finally, in the :guilabel:`Quantity` column, adjust the desired quantity of the product to sell.
+
+Once ready, click :guilabel:`Confirm` to confirm the sales order.
+
+Click the green :guilabel:`📈 (forecast graph)` icon on the product line to reveal the product's
+:guilabel:`Availability` tooltip. This tooltip reveals the reserved number of units for this order.
+Because the reservation method is set to *Before scheduled date*, the :guilabel:`Reserved` quantity
+reads `0 Units`.
+
+However, below that quantity reads `Available in stock`. This is because the quantity is available,
+but the scheduled date is five days from the order date. Since reservation isn't until two days
+before the scheduled delivery, it will not reserve the products until then.
```

Comment:
Yes, that is definitely helpful to clarify - I added in the following admonition to explain:

.. note::
   If there is **not** sufficient quantity of stock for the product included in the |SO|, the
   :guilabel:`📈 (forecast graph)` icon is red, instead of green.

   Instead of revealing the reserved number of units for the order, the :guilabel:`Availability`
   tooltip reads :guilabel:`Reserved`, and reveals the available number of units (e.g., `0 Units`).

   Additionally, unless there is a set replenishment or a live receipt, it also reads :guilabel:`No
   future availability`, in red text.

Suggestion implemented ✅